src/seedu/addressbook/commands/Command.java

```diff
@@ -16,6 +16,9 @@
     protected List<? extends ReadOnlyPerson> relevantPersons;
     private int targetIndex = -1;
 
+    protected static final boolean IS_MUTABLE = true;
+    protected static final boolean IS_IMMUTABLE = false;
```

Comment:
Would be alright to directly return the `boolean` value from the method rather than store it in a variable.